React/Views/RCTView.h

```diff
@@ -97,6 +100,10 @@ extern const UIAccessibilityTraits SwitchAccessibilityTrait;
 @property (nonatomic, assign) CGFloat borderStartWidth;
 @property (nonatomic, assign) CGFloat borderEndWidth;
 @property (nonatomic, assign) CGFloat borderWidth;
+// TODO: Implement logical border width logic
+@property (nonatomic, assign) CGFloat borderBlockWidth;
+@property (nonatomic, assign) CGFloat borderBlockEndWidth;
+@property (nonatomic, assign) CGFloat borderBlockStartWidth;
```

Comment:
I'll put up a follow-up PR implementing this but this is necessary here due to `RCT_VIEW_BORDER_PROPERTY` which creates set functions for both `border##SIDE##Color` and `border##SIDE##Width`